Sources/Widgets/Manipulators/LineManipulator/index.js

```diff
@@ -36,17 +36,34 @@ export function projectDisplayToLine(
 // ----------------------------------------------------------------------------
 
 function vtkLineManipulator(publicAPI, model) {
-  // Set our classNae
+  // Set our className
   model.classHierarchy.push('vtkLineManipulator');
 
+  // prettier-ignore
```

Comment:
why ignore ?

Reply:
It was to compact the function but it was unnecessary after all (prettier forbids `if (cond) { return val; }` but not `if (conf) return val;` )